db/structure.sql

```diff
@@ -4524,13 +3649,6 @@ CREATE UNIQUE INDEX index_flipper_features_on_key ON public.flipper_features USI
 CREATE UNIQUE INDEX index_flipper_gates_on_feature_key_and_key_and_value ON public.flipper_gates USING btree (feature_key, key, value);
 
 
---
--- Name: index_fs_dimensions_facility_id; Type: INDEX; Schema: public; Owner: -
---
-
-CREATE INDEX index_fs_dimensions_facility_id ON public.reporting_facility_state_dimensions USING btree (facility_id);
-
-
```

Comment:
I am not sure why after running `rails db:migrate` deleted this index.

Reply:
Found out that the new migration overrides the migration to add these deleted indexes. Updated migration file to add those indexes back.